src/mscorlib/shared/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1078,14 +1111,15 @@ internal struct Slot
                 public int SequenceNumber;
             }
         }
-    }
 
-    /// <summary>Padded head and tail indices, to avoid false sharing between producers and consumers.</summary>
-    [DebuggerDisplay("Head = {Head}, Tail = {Tail}")]
-    [StructLayout(LayoutKind.Explicit, Size = 3*Internal.PaddingHelpers.CACHE_LINE_SIZE)] // padding before/between/after fields
-    internal struct PaddedHeadAndTail
-    {
-        [FieldOffset(1*Internal.PaddingHelpers.CACHE_LINE_SIZE)] public int Head;
```

Comment:
Changing this from `CACHE_LINE_SIZE` to hardcoded 128 does not look like an improvement.

Otherplaces places still use `CACHE_LINE_SIZE`.